site.conf

```diff
@@ -29,10 +29,6 @@
     geo_location = {
       show_altitude = false,
     },
-    remote_login = {
```

Comment:
Should be included in Release Notes - and why.

Reply:
Why: this is a security risk
Release Notes: YES!!!

Reply:
> @margau https://git.darmstadt.ccc.de/ffda/site/blob/master/contrib/genpkglist.py this magic script is used for specific_site

Would suggest including the script in our site/contrib for transparency. Currently it's not clear where does it come from.